loadrunner_professional/README.md

```diff
@@ -0,0 +1,58 @@
+# LoadRunner Professional
+
+## Overview
+
+OpenText™ LoadRunner Professional provides a load testing solution that enables you to test the performance of diverse application types, and to identify and resolve issues before applications go live.
+
+LoadRunner Controller is a tool for creating and controlling LoadRunner Professional scenarios. A scenario defines the events that occur during each testing session. It controls the number of users to emulate (virtual users, or Vusers), the actions they perform, and the machines on which they run their emulations. You use scenarios to create load tests to check the reliability and strength of your servers.
+
+You can configure Controller to push real-time metrics and data from scenario runs to Datadog.
+
```

Comment:
Could you add another sentence or two here describing the metrics and data that's being sent in? i.e. is it performance data, is it metrics/logs/alerts/something else?

Reply:
i added the table from below